.github/release-please/config-backport.json

```diff
@@ -3,7 +3,7 @@
   "release-type": "node",
   "versioning": "always-bump-patch",
   "packages": {
-    ".": {
+    "apps/cli": {
```

Comment:
This way only changes in apps/cli/* will trigger release. Is that desirable?

Reply:
for now yeah, we don't want to release anything else. all other packages will be internal